sdk/identity/identity/src/constants.ts

```diff
@@ -31,7 +31,10 @@ export enum AzureAuthorityHosts {
   AzureChina = "https://login.chinacloudapi.cn",
   /**
    * Germany-based Azure Authority Host
-   */
+   *
+   * @deprecated Microsoft Cloud Germany was closed on October 29th, 2021.
+   *
+   * */
   AzureGermany = "https://login.microsoftonline.de",
```

Comment:
Won't this still show up in the enum? Will the users see it as deprecated when they get intellisense?

Reply:
Chatted offline Intellisense does a strikethrough on this. So we are good to go!

Reply:
![image](https://github.com/user-attachments/assets/0376b0e0-6fc8-46ff-a8e2-12dc3f0de1c0)